packages/react-components/react-table/stories/DataGrid/Default.stories.tsx

```diff
@@ -136,9 +136,9 @@ const columns: TableColumnDefinition<Item>[] = [
   }),
   createTableColumn<Item>({
     columnId: 'lastUpdate',
-    compare: (a, b) => {
-      return a.lastUpdate.label.localeCompare(b.lastUpdate.label);
-    },
+    // compare: (a, b) => {
+    //   return a.lastUpdate.label.localeCompare(b.lastUpdate.label);
+    // },
```

Comment:
Once the PR is ready, this should be returned to normal and maybe add a single column sortable story or show it in the sortable story.